watchdog/src/health.rs

```diff
@@ -0,0 +1,204 @@
+use crate::bitcoin_block_apis::BitcoinBlockApi;
+use crate::fetch::BlockInfo;
+use candid::CandidType;
+use serde::{Deserialize, Serialize};
+
+/// Status codes for the health of Bitcoin canister.
+#[derive(Clone, Debug, CandidType, PartialEq, Eq, Serialize, Deserialize)]
+pub enum StatusCode {
+    #[serde(rename = "no_data")]
+    NoData,
```

Comment:
I'm thinking that `undefined` or something along those lines would be more appropriate. Or `NotEnoughData`. Because the current one makes it seem like we couldn't get anything back but in the `HealthStatus` we might still see some (but not enough) data.